nix-darwin/default.nix

```diff
@@ -38,6 +39,24 @@ in
         <option>users.users.‹name?›.packages</option> option.
       '';
 
+      useGlobalPkgs = mkEnableOption ''
+        using the system configuration's <literal>pkgs</literal>
+        argument in Home Manager. This disables the Home Manager
+        options <option>nixpkgs.*</option>
+      '';
+
+      backupFileExtension = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "backup";
+        description = ''
+          On activation move existing files by appending the given
+          file extension rather than exiting with an error.
+        '';
+      };
+
+      verbose = mkEnableOption "verbose output on activation";
```

Comment:
This option does not seem to be used anywhere.

Reply:
Fixed